### PR TITLE
build(deps): bump github.com/cilium/ebpf from v0.9.1 to v0.12.3

### PR DESCRIPTION
**Pull request by wojiushixiaobai**
full diff: https://github.com/cilium/ebpf/compare/v0.9.1...v0.12.3

- Add support for loong64

### CODE REVIEW

File: vendor.mod [hunk 1]
```diff
@@ -101,7 +101,7 @@ require (
 	golang.org/x/mod v0.11.0
 	golang.org/x/net v0.17.0
 	golang.org/x/sync v0.3.0
-	golang.org/x/sys v0.13.0
+	golang.org/x/sys v0.14.1-0.20231108175955-e4099bfacb8c
```
Comment (thaJeztah) [inline vendor.mod:104]: It looks like v0.15.0 was tagged; also might be better to update this in a separate commit

File: vendor.mod [hunk 2]
```diff
@@ -131,7 +131,7 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/cenkalti/backoff/v4 v4.2.1 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
-	github.com/cilium/ebpf v0.9.1 // indirect
+	github.com/cilium/ebpf v0.12.3 // indirect
```
Comment (thaJeztah) [inline vendor.mod:134]: It looks like this is an indirect dependency, and used by `github.com/containerd/cgroups/v3`. Wondering if it should be updated there first (as well)

**Review by thaJeztah — COMMENTED** [review]
> Add support for loong64

If we want to verify that architecture to work, I guess we should also add it to the list of cross-compile targets as well; do you know if it currently works? Are other changes needed in the Dockerfile (and other places, such as the seccomp profile?)

Screenshot 2023-12-04 at 10 10 51

File: vendor.mod (hunk 2 above)
Comment (wojiushixiaobai) [reply vendor.mod:134]: Okay, I'll go take a look.
